### ISSUE TITLE
Items disappear from Sack/extended inventory and a Question

### ISSUE THREAD

**Issue by Tyranuss** [PrinceOfAmber/Cyclic#546]
**MC 1.10.2 and Cyclic 1.10.2-1.10.29**

**Single Player**

All of my items from Sacks and my extended inventory disappeared. It happened before from one of my sacks, but items were garbage and didn't pay much attention.
But now from all 6 of my Sacks and extended inventory... Now that's a problem...
A way to fix it?

Also a question about version 1.10.30. I've read on your page that you have cleared some config features, such as iron from silverfish,etc... 
Does that mean it's on this 1.10.2-1.10.30 version as well since you have updated it couple of days ago?

**Comment by PrinceOfAmber** [comment]
Do you have the server logs from when this occurred? or latest.log from single player?

I literally store those items in a flat file along with a backup, i know in single player its this folder 

/saves/whatever/playerdata/

and there you will see files, some with .info at the end and some with .backup (you may have to tell windows or whatever to show file extensions.

For me i have a file with the name '8364925c-2b2f-4391-b74c-a933e32c5e9f' because that is the id of my player.  So they still SHOULD exist somewhere. i hope . unless it saved and reloaded too many times and the empty wiped out the backup.

"Also a question about version 1.10.30. I've read on your page that you have cleared some config features, such as iron from silverfish,etc...
Does that mean it's on this 1.10.2-1.10.30 version as well since you have updated it couple of days ago?" Honestly I don't remember.  It should say in the config file if that option is listed.  

Also i am away this weekend but ill be looking at this on sunday after i get back

**Comment by PrinceOfAmber** [comment]
Just remembered, i keep an update file with all the changelogs.  The version that had those configs removed was 1.11.2-1.3.0 as shown here, so 1.10 still has them  https://github.com/PrinceOfAmber/Cyclic/blob/master/update.json#L159

**Comment by Tyranuss** [comment]
Thank you for answering my question.
I have found the said file, but I can't upload it, says id doesn't support that file type... Ehhh

**Comment by PrinceOfAmber** [comment]
Yeah the specific file probably wont get in. 

The best thing would be for me to get a log from the time the item loss happened.

Second best would be just any log file of you playing the game, that tells me what other mods are installed at the same time. its sometimes a compatibility issue or a server problem.

**Comment by Tyranuss** [comment]
Here are several logs from couple of days. Earliest I remember this issue happening (maybe it was 16th idk), and from 17th as well when I know item loss happened. 

[2017-10-15-3.log.gz](https://github.com/PrinceOfAmber/Cyclic/files/1408053/2017-10-15-3.log.gz)



[2017-10-17-1.log.gz](https://github.com/PrinceOfAmber/Cyclic/files/1408049/2017-10-17-1.log.gz)

[2017-10-18-1.log.gz](https://github.com/PrinceOfAmber/Cyclic/files/1408052/2017-10-18-1.log.gz)

**Comment by PrinceOfAmber** [comment]
Looks like something is wrong with the level, maybe it cant save or load?  level.dat is a vanilla minecraft file that stores lots of stuff, and in there it says

Exception reading C:\Users\User\AppData\Roaming\.minecraft\saves\A new begining #xn\level.dat
java.util.zip.ZipException: Not in GZIP format

Gonna keep digging/testing

**Comment by PrinceOfAmber** [comment]
When i search for "cyclic" in those files, i dont see any error messages that are /directly/ from my stuff.

I also did notice this

Couldn't look up profile properties for com.mojang.authlib.GameProfile@13c06b2b[id=00000000-0000-0000-0000-000000000000,name=SeekER,properties={}

Sounds like maybe it could not find your player file, or something like that? Normally your player name gets matched with a UUID, so for me, i never see that error and i get something thats not all zeroes

.......\saves\New World\playerdata\3e1fbeae-13f8-3fbb-9842-63bc739da918.dat

**Comment by Tyranuss** [comment]
Yeah, it was weird for me as well... I tried looking up my player file but all I saw were zeros... Idk maybe it could do something with me transfering to new PC and transporting my save files from old one via USB (could have viruses idk...) to my new PC... Eh I would help you more, but honestly even this is above my knowledge of PC. haha 
Thank you for your efforts. Who knows if it happens again, maybe to someone else we can find out more. Until then I'll just stay away from sacks and gladly use every other part of this beautiful mod. 
Thank you again. <3
